azure_functions_worker/dispatcher.py

```diff
@@ -262,6 +263,12 @@ async def _handle__worker_init_request(self, req):
                     'python version %s, worker version %s, request ID %s',
                     sys.version, __version__, self.request_id)
 
+        worker_init_request = req.worker_init_request
+        host_capabilities = worker_init_request.capabilities
```

Comment:
Nice. I think this is useful to be logged as well in the future. I'll take a note of this